src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -620,6 +620,35 @@
     ExtractToolPackage;
     CleanupRepo" />
 
+  <Target Name="Test"
+          Inputs="$(MSBuildProjectFullPath)"
+          Outputs="$(BaseIntermediateOutputPath)Test.complete">
```

Comment:
Why do we want this? You should be able to repeatedly execute tests easily.

Reply:
We have the same for the `Build` target. I'm fine with removing incremental support from this target.

Note that you can always disable msbuild's incremental behavior by passing the `--no-incremental` flag in.

Reply:
I feel that build and test should be treated separately with respect to incremental support and it wouldn't apply to test execution. FYI: Improving the UX of incremental builds is https://github.com/dotnet/source-build/issues/3666.